src/org/traccar/model/CalendarPermission.java

```diff
@@ -16,17 +16,7 @@
  */
 package org.traccar.model;
 
-public class CalendarPermission {
-
-    private long userId;
-
-    public long getUserId() {
-        return userId;
-    }
-
-    public void setUserId(long userId) {
-        this.userId = userId;
-    }
+public class CalendarPermission extends BaseUserPermission {
```

Comment:
We might need to adjust `QueryBuilder`, but that's fine if it helps to get rid of like 10 classes.